packages/client.test.generate/bin/generateMockOntology.mjs

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2025 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// @ts-check
+
+import { LegacyFauxFoundry } from "@osdk/shared.test";
+import { execa } from "execa";
+import { promises as fs, rmSync } from "node:fs";
+import { mkdir, readdir, writeFile } from "node:fs/promises";
+import { dirname, join } from "node:path";
+import { fileURLToPath } from "url";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+
+const argv = await yargs(hideBin(process.argv))
+  .usage("Usage: $0 --outDir [dir] [--version version]")
+  .option("outDir", {
+    alias: "o",
+    describe: "Output directory to generate the mock ontology",
+    type: "string",
+    demandOption: true,
+  })
+  .option("generatorVersion", {
+    alias: "v",
+    describe: "Version of the @osdk/generator package to use",
+    type: "string",
+  })
+  .help()
+  .argv;
+
+(async () => {
+  await generateMockOntologyAtDir(argv.outDir, argv.generatorVersion);
+})();
```

Comment:
FWIW, since this is an `mjs` file you can just do top level await, you dont have to make the IIfe